packages/router/api.js

```diff
@@ -0,0 +1,75 @@
+const express = require('express');
+const router = express.Router();
+const { pathMappingsData, loadPathMappings } = require('./pathmapping');
+const fsp = require("fs").promises;
+const path = require('path');
+const config = require("./config");
+
+const swaggerUi = require('swagger-ui-express');
+const YAML = require('yamljs');
+const openApiDocumentation = YAML.load('./openapi.yaml');
+
+
+
+router.get('/pathMappings', (req, res) => {
+    if (pathMappingsData.pathMappings.length === 0) {
+        res.json({ message: 'No mapping found' });
+    } else {
+        res.json(pathMappingsData.pathMappings);
+    }
+});
+
+
+
+router.get('/pathMappings/reload', async(req, res) => {
+    await loadPathMappings();
+    if (pathMappingsData.pathMappings.length === 0) {
+        res.json({ message: 'No mapping found' });
+    } else {
+        res.json(pathMappingsData.pathMappings);
+    }
+});
+
+
+
+router.post('/pathMappings', (req, res) => {
+    const { incoming_path, mapped_with } = req.body;
```

Comment:
I would recommend when you're using Pascal case throughout the project for all methods, apis, few variables, then we should continue with that for all the cases, and avoid the Snake case. In future you may consider it.

Reply:
Good catch, although I see camel not pascal @soumyadip007 

![Screenshot 2024-05-14 at 1 00 35 AM](https://github.com/spaship/spaship/assets/12994292/e822cdc3-6253-4bd9-9a5c-957cc1e82b52)